docs/example-external.md

```diff
@@ -20,5 +20,5 @@ sidebar_label: External Examples
   Tolinski
 
 <a href="https://youtu.be/JKOwJUM4_RM">
-  <img width="200px" alt="what is react testing library" src='https://img.youtube.com/vi/JKOwJUM4_RM/0.jpg' style="margin-left: 36px" />
+  <img width="200px" alt="what is react testing library" src='https://img.youtube.com/vi/JKOwJUM4_RM/0.jpg' style={{marginLeft: 36}} />
```

Comment:
What's up with this? Are the MD files being parsed as MDX now? This syntax is only valid for React.

Reply:
Yep, docusaurus 2 uses mdx, I will rename all md later on..

Reply:
I've added a task for this above so I'm resolving this. Thanks!